### ISSUE TITLE
package-info-cache warnings aren't super actionable yet

### ISSUE THREAD

**Issue by stefanpenner** [ember-cli/ember-cli#7964]
```
WARNING:
WARNING: /Users/spenner/l/ember-cli-pemberly-i18n_trunk/node_modules/@linkedin/ember-cli-pemberly-i18n/tests/dum
my/lib/in-repo-addon-test:
WARNING:    Missing package directory
WARNING:
WARNING: /Users/spenner/l/ember-cli-pemberly-i18n_trunk/node_modules/ember-source/node_modules/simple-html-token
izer/commands:
WARNING:    Missing package directory
WARNING:
WARNING: /Users/spenner/l/ember-cli-pemberly-i18n_trunk/node_modules/simple-html-tokenizer/commands:
WARNING:    Missing package directory
WARNING: For addon at path /Users/spenner/l/ember-cli-pemberly-i18n_trunk/node_modules/@linkedin/ember-cli-pemberly-i18n:
WARNING:    Excluding invalid/malformed/missing addon at relative path 'tests/dummy/lib/in-repo-addon-test'
```

**Comment by stefanpenner** [comment]
Resolution:

* fix words
* add ability to disable a specific warning via ENV var

**Comment by erichonkanen** [comment]
related? https://github.com/tildeio/simple-html-tokenizer/issues/66

**Comment by stefanpenner** [comment]
cc @dcombslinkedin

**Comment by dcombslinkedin** [comment]
@stefanpenner, what sort of wording do you think would be better (I agree that 'missing package directory' is pretty short and could be more explanatory). For the second one I can think of slightly better wording, but not by a lot. Regarding disabling specific warnings, would it not be better to just fix the errors instead? (For example, why is a test directory mentioned in the package.json as a path)?

**Comment by dcombslinkedin** [comment]
Please see https://github.com/ember-cli/ember-cli/pull/8058 for my updating of the wording and formatting (and removing what amount to duplicate errors for missing package directories, since they're referred to elsewhere). No attempt to provide any environment var to turn things off, but perhaps this is good enough?

**Comment by rwjblue** [comment]
closed by https://github.com/ember-cli/ember-cli/pull/8058

**Comment by rwjblue** [comment]
fix released in v3.4.3

**Comment by dcombslinkedin** [comment]
Awesome! Thanks, @rwjblue !
